### DISCUSSION TITLE
select into导出到本地文件，能指定机器吗？

### DISCUSSION THREAD

**Discussion by 731879846kk** [apache/doris#13818] [A - General / Q&A] [answered]
select into  outfile 这个导出CSV  file，能指定机器导出吗？  我目前是三个fe三个be，然后执行查询导出的时候，他都是在我be三台机器的任意一台进行导出，暂时没有设置并发导出。  这个是由哪个工作的be决定导出的位置吗？   因为我们这边需求查询出的文件需要上传。将来部署生产不能每次都还要去机器上找然后再上传吧。  所以想问问能不能指定 into file在固定机器上 （这个项目没有hadoop 暂时不导出到HDFS上）。

**Comment by LemonLiTree** [comment, marked as answer]
这个没办法，因为outfile方法首先会进行查询，生成的查询计划下发到哪台BE是doris决定的，直接导出到该BE上，所以没法手动指定，我刚试了一下，单副本单分区单分桶的情况下数据只会在一个BE上，这种情况outfile 只会在一台BE，注意：重新建表也会导致数据在不同BE，truncate不会。

**Reply by 731879846kk** [reply]
明白了，谢谢~   单副本这种也只能测试用，  那只能根据结果url去找文件了。
